schema.sql

```diff
@@ -18,8 +18,8 @@ CREATE TABLE "hosts" (
 --  https://www.debian.org/doc/debian-policy/ch-opersys.html#s9.2.2
 CREATE SEQUENCE user_id MINVALUE 4000 MAXVALUE 59999 NO CYCLE;
 
-CREATE DOMAIN username_t varchar(31) CHECK (
```

Comment:
the `getpwnam` is querying by the username.
`getpwnam        = SELECT username, passwd, gecos, homedir, shell, uid, gid FROM passwd_table WHERE username = $1``